qa/L0_infer_zero/test.sh

```diff
@@ -54,6 +54,7 @@ rm -f $SERVER_LOG $CLIENT_LOG
 rm -fr models && mkdir models
 cp -r /data/inferenceserver/${REPO_VERSION}/qa_identity_model_repository/* models/. && \
     cp -r /data/inferenceserver/${REPO_VERSION}/qa_ensemble_model_repository/qa_identity_model_repository/* models/.
+rm `find ./models/ -name '*onnx*'` -rf
```

Comment:
In terms of deletion I would recommend use embedded syntax for `find` utility: 
```
find ./models/ -name '*onnx*' -delete
```

Reply:
The above gave me:
```
+ find ./models0/ -name '*onnx*' -delete
find: cannot delete './models0/onnx_nobatch_sequence_object': Directory not empty
find: cannot delete './models0/onnx_nobatch_sequence_int32': Directory not empty
find: cannot delete './models0/onnx_nobatch_sequence_bool': Directory not empty
```
so I'll keep `rm ... ` for now

Reply:
I believe not all the time, but sometime. Since this PR will be reverted at some point, I'll stick to `rm ...`, but I'll keep in mind `find ... -delete` for future